knora/dsplib/models/user.py

```diff
@@ -689,7 +690,7 @@ def getAllUsersForProject(con: Connection, proj_shortcode: str) -> list[Any]:
 
         result = con.get(User.ROUTE)
```

Comment:
Now that I think about it, there is a route to get all users for a specific project, it's:
`/admin/projects/shortcode/{{project_shortcode}}/members`

It's probably easier to use this for `getAllUsersForProject` directly. There are also groups and projects for the user.